File: problems/p3783_mirror_distance_of_an_integer/test_solution.cpp
#include "solution.cpp"
#include <cassert>
#include <iostream>

int main() {
    Solution s;

    // Example 1
    assert(s.mirrorDistance(25) == 27);
    std::cout << "test_example_1 PASSED\n";

    // Example 2
    assert(s.mirrorDistance(10) == 9);
    std::cout << "test_example_2 PASSED\n";

    // Example 3
    assert(s.mirrorDistance(7) == 0);
    std::cout << "test_example_3 PASSED\n";

    std::cout << "\nAll tests passed!\n";
    return 0;
}

File: problems/p788_rotated_digits/test_solution.py
from .solution import Solution


def test_example_1():
    assert Solution().rotatedDigits(n=10) == 4


def test_example_2():
    assert Solution().rotatedDigits(n=1) == 0


def test_example_3():
    assert Solution().rotatedDigits(n=2) == 1
